js/photo-filters.js

```diff
@@ -0,0 +1,121 @@
+const EFFECTS = [
+  {
+    name: 'none',
+    style: 'none',
+    min: 0,
+    max: 100,
+    step: 1,
+    unit: '',
+  },
+  {
+    name: 'chrome',
+    style: 'grayscale',
+    min: 0,
+    max: 1,
+    step: 0.1,
+    unit: '',
+  },
+  {
+    name: 'sepia',
+    style: 'sepia',
+    min: 0,
+    max: 1,
+    step: 0.1,
+    unit: '',
+  },
+  {
+    name: 'marvin',
+    style: 'invert',
+    min: 0,
+    max: 100,
+    step: 1,
+    unit: '%',
+  },
+  {
+    name: 'phobos',
+    style: 'blur',
+    min: 0,
+    max: 3,
+    step: 0.1,
+    unit: 'px',
+  },
+  {
+    name: 'heat',
+    style: 'brightness',
+    min: 1,
+    max: 3,
+    step: 0.1,
+    unit: '',
+  },
+];
+
+const DEFAULT_EFFECT = EFFECTS[0];
+let chosenEffect = DEFAULT_EFFECT;
+
+const uploadImage = document.querySelector('.img-upload__preview');
+const effects = document.querySelector('.effects');
+const slider = document.querySelector('.effect-level__slider');
+const sliderContainer = document.querySelector('.img-upload__effect-level');
+const levelEffect = document.querySelector('.effect-level__value');
+
+const isDefault = () => chosenEffect === DEFAULT_EFFECT;
+
+noUiSlider.create(slider, {
+  range: {
+    min: DEFAULT_EFFECT.min,
+    max: DEFAULT_EFFECT.max,
+  },
+  start: DEFAULT_EFFECT.max,
+  step: DEFAULT_EFFECT.step,
+  connect: 'lower',
+});
+
+const showSlider = () => {
+  sliderContainer.classList.remove('hidden');
+};
+
+const hideSlider = () => {
+  sliderContainer.classList.add('hidden');
+};
+
+const updateSlider = () => {
+  slider.noUiSlider.updateOptions({
+    range: {
+      min: chosenEffect.min,
+      max: chosenEffect.max,
+    },
+    step: chosenEffect.step,
```

Comment:
По ТЗ при переключении ещё нужно устанавливать в максимальное значение положение слайдера
```suggestion
    start: chosenEffect.max,
```